source/ShopPanel.h

```diff
@@ -31,6 +32,8 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #include <string>
 #include <vector>
 
+using namespace std;
+
```

Comment:
We don't put this in header files. (And it doesn't even look like it's being used.)
```suggestion
```